rise/static/main.js

```diff
@@ -526,13 +627,18 @@ define([
                 // turn off reveal native help
                 help: false,
 
+                // key bindings configurable by nbconfig  are defined below - 
+                // this shall only be used for hard wired bindings, which should 
+                // not be adjustable by user settings.
                 keyboard: {
                   13: null, // Enter disabled
                   27: null, // ESC disabled
+                  35: null, // End - last slide disabled (will be set in custom keys)
+                  36: null, // Home - first slide disabled (will be set in custom keys)
                   38: null, // up arrow disabled
                   40: null, // down arrow disabled
                   66: null, // b, black pause disabled, use period or forward slash
-                  70: fullscreenHelp, // disable fullscreen inside the slideshow, makes codemirror unreliable
+                  // 70: fullscreenHelp, // disable fullscreen inside the slideshow, makes codemirror unreliable
```

Comment:
IIRC the initial purpose of use redefining that 70 key - I guess it's 'f' or something - is to disable the native 'fullscreen' reveal feature 
(to be honest it was disabled a long time ago with an older version of reveal; and I'm not sure if it's still a problem or not, but that's another thing entirely of course)
so IMHO it would be safer to keep `70: null` in this list, to be sure to disable that binding, even if the user decides to map that function to another key

Reply:
I was also a bit confused by this action... But again I agree - this list should contain only disabling of bindings and default/custom bindings will be assigned in the next step.